pkg/ui/console/client.go

```diff
@@ -150,15 +150,15 @@ func (c Client) DownloadPolicy(ctx context.Context, args []string) error {
 }
 
 func (c Client) RunPolicies(ctx context.Context, args []string, policyName, outputDir, subPath string, stopOnFailure, skipVersioning, failOnViolation, noResults bool) error {
-	c.c.Logger.Debug("Received params: args: %v, policyName: %s, outputDir: %s, stopOnFailure: %v, skipVersioning: %v, failOnViolation: %v, noResults: %v", args, policyName, outputDir, stopOnFailure, skipVersioning, failOnViolation, noResults)
+	c.c.Logger.Debug("Received params:", "args", args, "policyName", policyName, "outputDir", outputDir, "stopOnFailure", stopOnFailure, "skipVersioning", skipVersioning, "failOnViolation", failOnViolation, "noResults", noResults)
 
 	policiesToRun, err := client.FilterPolicies(args, c.cfg.Policies, policyName, subPath)
 
 	if err != nil {
 		ui.ColorizedOutput(ui.ColorError, err.Error())
 		return err
 	}
-	c.c.Logger.Info("Policies to run: %v", policiesToRun)
+	c.c.Logger.Info("Policies to run", "policies", policiesToRun)
```

Comment:
I changed this as well in my PR #309, policiesToRun needs a better string function since its prints a list of structs it should only print the list of names.